app/controllers/actions/RequiredUserAnswers.scala

```diff
@@ -53,47 +53,69 @@ class RequiredUserAnswers @Inject()(genericLogger: GenericLogger)(implicit val e
+      RequiredPage(ContractPackingPage, List.empty)(implicitly[Reads[Boolean]]),
       RequiredPage(HowManyContractPackingPage,
-        Some(PreviousPage(ContractPackingPage, List(true))(implicitly[Reads[Boolean]])))(implicitly[Reads[LitresInBands]]),
-      RequiredPage(ImportsPage, Option.empty)(implicitly[Reads[Boolean]]),
+        List(PreviousPage(ContractPackingPage, List(true))(implicitly[Reads[Boolean]])))(implicitly[Reads[LitresInBands]]),
+      RequiredPage(ImportsPage, List.empty)(implicitly[Reads[Boolean]]),
       RequiredPage(HowManyImportsPage,
-        Some(PreviousPage(ImportsPage, List(true))(implicitly[Reads[Boolean]])))(implicitly[Reads[LitresInBands]]),
-      RequiredPage(StartDatePage, Option.empty)(implicitly[Reads[LocalDate]]),
-      RequiredPage(PackAtBusinessAddressPage, Option.empty)(implicitly[Reads[Boolean]]),
-      RequiredPage(PackagingSiteDetailsPage, Option.empty)(implicitly[Reads[Boolean]]),
-      RequiredPage(AskSecondaryWarehousesPage, Option.empty)(implicitly[Reads[Boolean]]),
+        List(PreviousPage(ImportsPage, List(true))(implicitly[Reads[Boolean]])))(implicitly[Reads[LitresInBands]]),
+      RequiredPage(StartDatePage, List.empty)(implicitly[Reads[LocalDate]]),
+      RequiredPage(PackAtBusinessAddressPage, List(
+        PreviousPage(HowManyLitresGloballyPage, List(HowManyLitresGlobally.enumerable.withName("small").get,
+            HowManyLitresGlobally.enumerable.withName("xnot").get))(implicitly[Reads[HowManyLitresGlobally]]),
+        PreviousPage(ContractPackingPage, List(true))(implicitly[Reads[Boolean]])))(implicitly[Reads[Boolean]]),
+      RequiredPage(PackAtBusinessAddressPage, List(
+        PreviousPage(HowManyLitresGloballyPage, List(HowManyLitresGlobally.enumerable.withName("large").get))(implicitly[Reads[HowManyLitresGlobally]]),
+        PreviousPage(OperatePackagingSitesPage, List(false))(implicitly[Reads[Boolean]]),
+        PreviousPage(ContractPackingPage, List(true))(implicitly[Reads[Boolean]])))(implicitly[Reads[Boolean]]),
+      RequiredPage(PackAtBusinessAddressPage, List(
+        PreviousPage(HowManyLitresGloballyPage, List(HowManyLitresGlobally.enumerable.withName("large").get))(implicitly[Reads[HowManyLitresGlobally]]),
+        PreviousPage(OperatePackagingSitesPage, List(true))(implicitly[Reads[Boolean]]),
+        PreviousPage(ContractPackingPage, List(true))(implicitly[Reads[Boolean]])))(implicitly[Reads[Boolean]]),
+      RequiredPage(PackAtBusinessAddressPage, List(
+        PreviousPage(HowManyLitresGloballyPage, List(HowManyLitresGlobally.enumerable.withName("large").get))(implicitly[Reads[HowManyLitresGlobally]]),
+        PreviousPage(OperatePackagingSitesPage, List(true))(implicitly[Reads[Boolean]]),
+        PreviousPage(ContractPackingPage, List(false))(implicitly[Reads[Boolean]])))(implicitly[Reads[Boolean]]),
+      RequiredPage(PackagingSiteDetailsPage,
```

Comment:
can these be condensed into 1?
```suggestion
     RequiredPage(PackAtBusinessAddressPage, List(
        PreviousPage(HowManyLitresGloballyPage, List(HowManyLitresGlobally.enumerable.withName("large").get))(implicitly[Reads[HowManyLitresGlobally]]),
        PreviousPage(OperatePackagingSitesPage, List(true))(implicitly[Reads[Boolean]]),
        PreviousPage(ContractPackingPage, List(true, false))(implicitly[Reads[Boolean]])))(implicitly[Reads[Boolean]]),
```

Reply:
Suggestion added